api/backend/blueflood/blueflood.go

```diff
@@ -88,7 +88,7 @@ func (b *Blueflood) FetchSeries(metric api.TaggedMetric, predicate api.Predicate
 	case api.SampleMax:
 		selectResultField = "Max"
 	default:
-		return nil, errors.New(fmt.Sprintf("Unsupported SampleMethod %s", sampleMethod))
+		return nil, errors.New(fmt.Sprintf("Unsupported SampleMethod %d", sampleMethod))
```

Comment:
`go vet` pointed out this formatting error (`sampleMethod` is a synonym for `int` and not `string`).